xgi/core/simplicialcomplex.py

```diff
@@ -906,3 +906,29 @@ def is_frozen(self):
             return self.frozen
         except AttributeError:
             return False
+
+    def k_skeleton(self, order, in_place=True):
+        """Returns the k-skeleton of the simplicial complex.
+
+        Parameters
+        ----------
+        order : int
+            The order (k) of the skeleton to return.
+        in_place : bool, optional
+            Whether to modify the current hypergraph or output a new one, by default
+            True.
+        """
+        if in_place:
+            _H = self
+        else:
+            _H = self.copy()
+        max_order = max(len(edge) for edge in _H._edge.values()) - 1
```

Comment:
```suggestion
        max_order = xgi.max_edge_order(_H)
```

Reply:
I don't think that I can do `from ..algorithms.properties import max_edge_order` in this module because this would lead to a circular import... this probably hints in the direction of having the function in the `convert` module then

Reply:
```
xgi/tests/conftest.py:6: in <module>
    import xgi
xgi/xgi/__init__.py:1: in <module>
    from . import (
xgi/xgi/core/__init__.py:4: in <module>
    from .simplicialcomplex import SimplicialComplex
xgi/xgi/core/simplicialcomplex.py:19: in <module>
    from ..algorithms.properties import max_edge_order
xgi/xgi/algorithms/__init__.py:1: in <module>
    from . import properties, assortativity, centrality, clustering, connected
xgi/xgi/algorithms/centrality.py:10: in <module>
    from ..convert import to_line_graph
xgi/xgi/convert/__init__.py:1: in <module>
    from . import (
xgi/xgi/convert/bipartite_graph.py:7: in <module>
    from ..generators import empty_hypergraph
xgi/xgi/generators/__init__.py:1: in <module>
    from . import (
xgi/xgi/generators/simplicial_complexes.py:15: in <module>
    from ..core import SimplicialComplex
```